test/benchmark/main.cc

```diff
@@ -3,14 +3,32 @@
 #include "test/test_common/environment.h"
 
 #include "benchmark/benchmark.h"
+#include "tclap/CmdLine.h"
 
-// Boilerplate main(), which discovers benchmarks and runs them.
+bool g_skip_expensive_benchmarks = false;
```

Comment:
Can we make this a global function rather than a global variable, e.g.

static bool skip_expensive_benchmarks = false;
bool SkipExpensiveBenchmarks() { return skip_expensive_benchmarks; }

you'd then declare it in a new main.h, I guess, which benchmarks would need to include to switch on this.

@antoniovicente WDYT? Is there a better way to do this?

Reply:
done.